src/LazyJSON.jl

```diff
@@ -1,5 +1,7 @@
 module LazyJSON
 
+using PropertyDicts
```

Comment:
Matches original `using` call:
```suggestion
using PropertyDicts: PropertyDict
```